web/libs/ui/src/assets/icons/keypoint-disabled.svg

```diff
@@ -1,6 +1,6 @@
 <svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
 <g opacity="0.2">
-<rect x="8.10049" y="18" width="14" height="14" rx="2" transform="rotate(-45 8.10049 18)" fill="black"/>
+<rect x="8.10049" y="18" width="14" height="14" rx="2" transform="rotate(-45 8.10049 18)" fill="currentColor"/>
```

Comment:
@ricardoantoniocm this icon uses 2 colors, with swap out of black towards currentColor this could have issues in light mode with white on `currentColor` applied. can you suggest an updated icon that would work in both light and dark mode for this? we should look at addressing this in a separate ticket as its not directly related to Button.
@nick-skriabin great catch on this - but I'm concerned that we are addressing too much in this PR